easybuild/framework/easyblock.py

```diff
@@ -1438,6 +1438,26 @@ def make_module_extra(self, altroot=None, altversion=None):
                                      value, type(value))
             lines.append(self.module_generator.append_paths(key, value, allow_abs=self.cfg['allow_append_abs_path']))
 
+        # Add automatic PYTHONPATH or EBPYTHONPREFIXES if they aren't already present and python paths exist
+        if not os.path.isfile(os.path.join(self.installdir, 'bin/python')):  # only needed when not a python install
+            python_paths = [path for path in glob.glob('lib/python*/site-packages', root_dir=self.installdir)
```

Comment:
`root_dir` is Python 3.10: https://docs.python.org/3/library/glob.html#glob.glob